cost/aws/old_snapshots/aws_delete_old_snapshots.pt

```diff
@@ -251,7 +251,7 @@ datasource "ds_get_snapshot_ami" do
       field "imageId", xpath(col_item, "imageId")
       field "region", val(iter_item, "region")
       field "snapshotId", val(iter_item, "snapshotId")
-      field "tagKeyValue", val(iter_item, "tagKeyValue")
+      field "tags", val(iter_item, "tags")
```

Comment:
Have you tested that the `tags` field shows up in the XML with exactly the same format as `tagKeyValue` did? It seems odd that AWS would have changed the format of the `2016-11-15` version of the `DescribeImages` response.